Modules/Devices/src/Devices.Domain/Aggregates/PushNotifications/PnsRegistration.cs

```diff
@@ -7,6 +7,7 @@ namespace Backbone.Modules.Devices.Domain.Aggregates.PushNotifications;
 
 public class PnsRegistration
 {
+    // ReSharper disable once UnusedMember.Local
```

Comment:
Why the comment?

Reply:
...one final warning is that the empty constructor is never used - hence suppressing it with the comment.

Reply:
It's not null-related, but okay. ;)